### ISSUE TITLE
Export lz-analyze to Leela 0.11.0

### ISSUE THREAD

**Issue by kaorahi** [leela-zero/leela-zero#2631]
I exported lz-analyze in Leela Zero to Leela 0.11.0 so that it works on Lizzie etc. Please let me know if I misunderstand anything.

https://github.com/kaorahi/Leela/tree/lz-analyze

**Comment by CGLemon** [comment]
Very Good Job.
But A very little problem is there.
If we disable the neural network, the opening book will be no effect in analyze mode.
The 19x19 board is much weaker at beginning in neural network disability case.

**Comment by kaorahi** [comment]
I'm satisfied with the default setting of Leela and never use `--nonets`. So I have little motivation on this problem unfortunately.

**Comment by CGLemon** [comment]
Do you have any plan to export another functions to Leela 0.11.0?
Likely the Leela Zero network or others.

**Comment by kaorahi** [comment]
No. This is just a quick hack and I'm satisfied with it. So you may prefer forking if you are interested in further developments. ;)

**Comment by pva22** [comment]
> I exported lz-analyze in Leela Zero to Leela 0.11.0 so that it works on Lizzie etc. Please let me know if I misunderstand anything.
> 
> https://github.com/kaorahi/Leela/tree/lz-analyze

Please explain what this command does? Where can I see a description of the commands and an explanation of their conclusions?

**Comment by kaorahi** [comment]
KataGo's document may be useful for you.

https://github.com/lightvector/KataGo/blob/master/docs/GTP_Extensions.md
